stylus/settings/breakpoints.styl

```diff
@@ -158,12 +158,25 @@ no-mobile()
     @media (min-width: (size-helper('min', small)/basefont)rem)
         {block}
 
+/*
+    no-tablet()
+
+    Refers to (min-width: 1024px)
+
+    Weight: 4
+
+    Styleguide Settings.breakpoints.no-tablet
+*/
+no-tablet()
```

Comment:
`larger-than-tablet` seems clearer to me, I was expecting this breakpoint to match (width < small) || (width > medium).

Reply:
or `gt-tablet` (gt for greater than)

Reply:
yeah… I keep doubting about the naming. I like your ideas though.